app/assets/javascripts/application.js

```diff
@@ -73,6 +73,7 @@ function onContentLoad(){
       if ($(this).data('on-complete')){
         _.get(window, $(this).data('on-complete')).call(null, this, status);
       }
+      tfm.tools.activateTooltips(this)
```

Comment:
maybe just default to loading on content load (instead of null in the call function above)?

Reply:
The function above is a user-defined hook - each consumer can define a method to call on complete. I don't see a reason why tooltips code should be broken for AJAX, unless the user specifies a hook explicitly.

Reply:
this might lead to activeTooltips running multiple times, but I don't think that should block the pr. :+1: from my side.